src/screens/AnonymousTicketPurchase/components/Actions.tsx

```diff
@@ -0,0 +1,210 @@
+import Button from '@atb/components/button';
+import AnonymousTicketPurchases from '@atb/translations/screens/subscreens/AnonymousTicketPurchases';
+import React from 'react';
+import {useFinishOnboarding} from '@atb/screens/Onboarding/use-finish-onboarding';
+import {useTranslation} from '@atb/translations';
+import {StyleSheet, useTheme} from '@atb/theme';
+import Bugsnag from '@bugsnag/react-native';
+import {useMobileTokenContextState} from '@atb/mobile-token/MobileTokenContext';
+import {useAuthState} from '@atb/auth';
+import useIsLoading from '@atb/utils/use-is-loading';
+import {ActivityIndicator, View} from 'react-native';
+
+export type CallerRoute =
+  | 'ConsequencesFromOnboarding'
+  | 'ConsequencesFromLogout'
+  | 'ConsequencesFromTicketPurchase'
+  | 'ConsequencesFromLoginOnboarding';
+
+const Actions = ({
+  callerRoute,
+  navigation,
+}: {
+  callerRoute: CallerRoute;
+  navigation: any;
+}) => {
+  const {t} = useTranslation();
+  const finishOnboarding = useFinishOnboarding();
+  const {wipeToken} = useMobileTokenContextState();
+  const {signOut} = useAuthState();
+  const [isLoading, setIsLoading] = useIsLoading(false);
+
+  const navigateToTickets = () => {
+    navigation?.navigate('Ticketing');
+  };
+
+  const navigateTologIn = async () => {
+    navigation?.navigate('LoginInApp', {
+      screen: 'PhoneInputInApp',
+      params: {
+        afterLogin: {
+          routeName: 'TabNavigator',
+        },
+      },
+    });
+  };
+
+  const logout = async () => {
+    setIsLoading(true);
+    await logoutAndWipeTokens();
```

Comment:
Can `signOut` fail here? Causing `setIsLoading(false)` never to be set?

Reply:
mmm might be..we are wiping tokens and then signing as anonymous.  I can put it inside a try-catch and set an error if it fails.